cmd/in-toto/record.go

```diff
@@ -138,6 +138,14 @@ command is executed. Symlinks are followed.`,
 are stored in the resulting link metadata after the
 command is executed. Symlinks are followed.`,
 	)
+
+	recordCmd.Flags().BoolVarP(
+		&lineNormalization,
+		"line-normalization",
```

Comment:
```suggestion
		"normalize-line-endings",
```

IMO this reads better as a flag. I'm also not sure if we want to include a short flag for this one.